search/templates/cloudplatform_pcs_policy.tpl

```diff
@@ -12,6 +12,15 @@
                 "es:ESHttpPut"
             ],
             "Resource":"arn:aws:es:${region}:${account_id}:domain/${domain}" 
+        },
+        {
+            "Sid": "CrossIAMPermissionsRead",
+            "Effect": "Allow",
+            "Action": [
+                "es:Describe*",
+                "es:List*"
+            ],
+            "Resource": "arn:aws:es:eu-west-2:723123699647:domain/*"
```

Comment:
Can this account number be from a variable instead as I've no idea what it belongs to

Reply:
updated to use account_id